src/skeleton/read.rs

```diff
@@ -89,14 +96,62 @@ pub(super) fn manifests<P: AsRef<Path>>(
                 &absolute_path
             )
         })?;
+        let mut targets = package.map(|p| gather_targets(p)).unwrap_or_default();
+        targets.sort_by(|a, b| a.path.cmp(&b.path));
```

Comment:
We can use a `BTreeSet` here as well.

Reply:
By implementing `Ord` for `Target`? I'm not sure if we want it to be orderable by path everywhere. Otherwise we'd need to wrap it in another newtype.

Reply:
What's the downside? They are all private implementation details of `cargo-chef`, nothing here is offered as a library to anyone else.

Reply:
And, same thing, it doesn't need to be sorted by path. It just needs to be sorted for determinism.